engine/action_contact.go

```diff
@@ -70,7 +72,7 @@ func (c *Client) AddContactRequest(ctx context.Context, fullName, paymailAdress,
 		return nil, err
 	}
 
-	save := false
+	var save bool
```

Comment:
Why?

Reply:
Linter was complaining about this because we're setting default value for the bool, and that value is always assigned in lines below, in other words, we don't use this default.